src/app/store.ts

```diff
@@ -1,6 +1,9 @@
 import { combineSlices, configureStore } from '@reduxjs/toolkit';
+import { todosSlice } from '../features/todos';
+import { currentTodoSlice } from '../features/currentTodo';
+import { filterSlice } from '../features/filter';
 
-const rootReducer = combineSlices();
+const rootReducer = combineSlices(todosSlice, currentTodoSlice, filterSlice);
```

Comment:
The function `combineSlices` is incorrect. You should use `combineReducers` from `@reduxjs/toolkit` to combine the slices: `const rootReducer = combineReducers({ todos: todosSlice.reducer, currentTodo: currentTodoSlice.reducer, filter: filterSlice.reducer });`.